README.md

```diff
@@ -1,3 +1,373 @@
+
+```
+export NEPTUNE_API_TOKEN="h0dHBzOi8aHR0cHM.4kl0jvYh3Kb8...ifQ=="
+```
+
+```
+export NEPTUNE_PROJECT="team-alpha/project-x"
+```
+
+You're ready to start using Neptune Scale.
+
+## Example usage
+
+```python
+from neptune_scale import Run
+
+run = Run(
+    family="RunFamilyName",
+    run_id="SomeUniqueRunIdentifier",
+)
+
+run.log_configs(
+    data={"learning_rate": 0.001, "batch_size": 64}
+)
+
+# inside a training loop
+for e in epoch:
+    run.log_metrics(
+        data={"acc": 0.89, "loss": 0.17},
+    )
+
+run.add_tags(tags=["tag1", "tag2"])
+
+run.close()
+```
+
+## API reference
+
+### `Run`
+
+Representation of experiment tracking metadata logged with Neptune Scale.
+
+#### Initialization
+
+Initialize with the class constructor:
+
+```python
+from neptune_scale import Run
+
+run = Run(...)
+```
+
+or using a context manager:
+
+```python
+from neptune_scale import Run
+
+with Run(...) as run:
+    ...
+```
+
+__Parameters__
+
+| Name             | Type             | Default | Description                                                               |
+|------------------|------------------|---------|---------------------------------------------------------------------------|
+| `family`         | `str`            | -       | Identifies related runs. All runs of the same lineage must have the same `family` value. That is, forking is only possible within the same family. Max length: 128 bytes. |
+| `run_id`         | `str`            | -       | Identifier of the run. Must be unique within the project. Max length: 128 bytes. |
+| `project`        | `str`, optional  | `None`  | Name of a project in the form `workspace-name/project-name`. If `None`, the value of the `NEPTUNE_PROJECT` environment variable is used. |
+| `api_token`      | `str`, optional  | `None`  | Your Neptune API token or a service account's API token. If `None`, the value of the `NEPTUNE_API_TOKEN` environment variable is used. To keep your token secure, don't place it in source code. Instead, save it as an environment variable. |
+| `resume`         | `bool`, optional | `False` | If `False` (default), creates a new run. To continue an existing run, set to `True` and pass the ID of an existing run to the `run_id` argument. To fork a run, use `fork_run_id` and `fork_step` instead. |
+| `mode`           | `Literal`, `"async"` or `"disabled"` | `"async"` | Mode of operation. If set to `"disabled"`, the run doesn't log any metadata. |
+| `experiment_name`  | `str`, optional  | `None` | Name of the experiment to associate the run with. Learn more about [experiments](https://docs-beta.neptune.ai/experiments) in the Neptune documentation. |
+| `creation_time`  | `datetime`, optional | `None` | Custom creation time of the run. |
+| `fork_run_id`    | `str`, optional  | `None` | If forking off an existing run, ID of the run to fork from. |
```

Comment:
I think that `ID of the run to fork from` should be enough. You always have to fork from an existing run.

Reply:
```suggestion
| `fork_run_id`    | `str`, optional  | `None` | The ID of the run to fork from. |
```